pkgs/firedragon/src.json

```
@@ -0,0 +1,15 @@
+{
+  "packageVersion": "111.0.1-1",
+  "source": {
+    "rev": "111.0.1-1",
+    "sha256": "045nisl000ll0pzir9zhrkbbkl87bsd38mygx7gz9kv6p0pppl7i"
+  },
+  "firedragon": {
+    "rev": "3a9d7188b9455bcc1fd8e713d79ff9bb3ffb32da",
+    "sha256": "BKK7ZEqf+rNK0p41dcpn8hqGRLAKxxr4kMCjcEmAeYA="
```

```suggestion
    "hash": "sha256-BKK7ZEqf+rNK0p41dcpn8hqGRLAKxxr4kMCjcEmAeYA="
```

(this one was already in the correct base)